cloud-aws/src/main/java/com/sequenceiq/cloudbreak/cloud/aws/connector/resource/AwsLaunchService.java

```diff
@@ -249,10 +254,19 @@ private void updateCloudformationWithLoadBalancers(AuthenticatedContext ac, Clou
         }
     }
 
-    private AwsLoadBalancer convert(CloudLoadBalancer cloudLoadBalancer, CloudStack stack, List<CloudResource> instances) {
+    private AwsLoadBalancerScheme determinePublicVsPrivateSchema(AwsNetworkView awsNetworkView, AmazonEC2Client amazonEC2Client) {
+        String subnetId = awsNetworkView.getExistingSubnet();
```

Comment:
are we sure it's not a subnet list here? shall we validate?

Reply:
In debugging, I have only seen 1 subnet get passed in when I select 3 during env creation.

Reply:
Hreeve is working on a change for semi-private networks to support a list of subnets.